guide/src/class.md

```diff
@@ -121,6 +121,27 @@ created from Rust, but not from Python.
 
 For arguments, see the [`Method arguments`](#method-arguments) section below.
 
+### Constructors which accept a class argument
```

Comment:
I think I'd prefer to add this section as part of the `#[classmethod]` section later; the way this document currently reads this is both the documentation of `#[new]` but also a "getting started with classes". I think showing users the option of `#[classmethod]` here may be jumping the gun?